package.json

```diff
@@ -20,10 +20,13 @@
   "dependencies": {
     "@glimmer/component": "^0.2.0",
     "@glimmer/di": "^0.1.9",
+    "@glimmer/runtime": "^0.22.0",
```

Comment:
We directly depend on this stuff but never included it in the `package.json`. I discovered this because I was getting version mismatches.